README.md

```diff
@@ -32,143 +38,191 @@ Rouge::Themes::Base16.mode(:light).render(scope: '.highlight')
+encouraged to write your own formatter if you need something custom.
+
+The built-in formatters are:
 
-#### Formatters
+* `Rouge::Formatters::HTML.new` will render your code with standard class names
+  for tokens, with no div-wrapping or other bells or whistles.
 
-As of Rouge 2.0, you are encouraged to write your own formatter for custom formatting needs.
-Builtin formatters include:
+* `Rouge::Formatters::HTMLInline.new(theme)` will render your code with no class
+  names, but instead inline the styling options into the `style=` attribute.
+  This is good for emails and other systems where CSS support is minimal.
 
-* `Rouge::Formatters::HTML.new` - will render your code with standard class names for tokens,
-  with no div-wrapping or other bells or whistles.
-* `Rouge::Formatters::HTMLInline.new(theme)` - will render your code with no class names, but
-  instead inline the styling options into the `style=` attribute. This is good for emails and
-  other systems where CSS support is minimal.
-* `Rouge::Formatters::HTMLLinewise.new(formatter, class: 'line-%i')`
-  This formatter will split your code into lines, each contained in its own div. The
-  `class` option will be used to add a class name to the div, given the line
-  number.
-* `Rouge::Formatters::HTMLLineTable.new(formatter, opts={})` will output an HTML table containing
-  numbered lines, each contained in its own table-row. Options are:
+* `Rouge::Formatters::HTMLLinewise.new(formatter, class: 'line-%i')` will split
+  your code into lines, each contained in its own div. The `class` option will
+  be used to add a class name to the div, given the line number.
+
+* `Rouge::Formatters::HTMLLineTable.new(formatter, opts={})` will output an HTML
+  table containing numbered lines, each contained in its own table-row. Options
+  are:
     * `start_line: 1` - the number of the first row
-    * `line_id: 'line-%i'` - a `sprintf` template for `id` attribute with current line number
+    * `line_id: 'line-%i'` - a `sprintf` template for `id` attribute with
+      current line number
     * `line_class: 'lineno'` - a CSS class for each table-row
     * `table_class: 'rouge-line-table'` - a CSS class for the table
     * `gutter_class: 'rouge-gutter'` - a CSS class for the line-number cell
     * `code_class: 'rouge-code'` - a CSS class for the code cell
-* `Rouge::Formatters::HTMLPygments.new(formatter, css_class='codehilite')`
-  wraps the given formatter with div wrappers generally expected by stylesheets designed for
-  Pygments.
-* `Rouge::Formatters::HTMLTable.new(formatter, opts={})` will output an HTML table containing
-  numbered lines. Options are:
+
+* `Rouge::Formatters::HTMLPygments.new(formatter, css_class='codehilite')` wraps
+  the given formatter with div wrappers generally expected by stylesheets
+  designed for Pygments.
+
+* `Rouge::Formatters::HTMLTable.new(formatter, opts={})` will output an HTML
```

Comment:
You may also want to mention how this is different to the line-table formatter above.
Currently, its not obviously discernible.

Reply:
Can you suggest some wording here? I'm not confident enough accurately describing the differences.

Reply:
```
`Rouge::Formatters::HTMLTable.new(formatter, opts={})` will output an HTML table
containing numbered lines similar to `Rouge::Formatters::HTMLLineTable`, except that
the table from this formatter has just a single table-row. Therefore, while the table
is more JavaScript DOM-friendly, long code lines will mess with the column alignment.
```

Reply:
Thanks! 👍